src/__swaps__/screens/Swap/utils/assets.ts

```diff
@@ -0,0 +1,347 @@
+import { AddressZero } from '@ethersproject/constants';
```

Comment:
this is all cool rn but were gonna need to merge with our current parsers/utils at some point or things are going to get real confusing